include/mbedtls/platform_util.h

```diff
@@ -215,12 +215,12 @@ void *mbedtls_platform_memcpy( void *dst, const void *src, size_t num );
 int mbedtls_platform_memmove( void *dst, const void *src, size_t num );
 
 /**
- * \brief       Secure memcmp
+ * \brief       Secure check if the buffers have the same data.
  *
- *              This is a constant-time version of memcmp(). If
- *              MBEDTLS_ENTROPY_HARDWARE_ALT is defined, the order is also
- *              randomised using the hardware RNG in order to further harden
- *              against side-channel attacks.
+ *              This is a constant-time version of memcmp(), but without checking
```

Comment:
There was no API warranty that there was a check if the bytes are greater or lower before. The return code specifies only two possible return values.